sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -224,16 +221,15 @@ Run a predictive model to identify a collection of entities found in the passed-
 ```C# Snippet:RecognizeLinkedEntities
 string input = "Microsoft was founded by Bill Gates and Paul Allen.";
 
-Response<IReadOnlyCollection<LinkedEntity>> response = client.RecognizeLinkedEntities(input);
-IEnumerable<LinkedEntity> linkedEntities = response.Value;
+IReadOnlyCollection<LinkedEntity> linkedEntities = client.RecognizeLinkedEntities(input).Value;
 
-Console.WriteLine($"Extracted {linkedEntities.Count()} linked entit{(linkedEntities.Count() > 1 ? "ies" : "y")}:");
+Console.WriteLine($"Extracted {linkedEntities.Count} linked entit{(linkedEntities.Count > 1 ? "ies" : "y")}:");
 foreach (LinkedEntity linkedEntity in linkedEntities)
 {
     Console.WriteLine($"Name: {linkedEntity.Name}, Language: {linkedEntity.Language}, Data Source: {linkedEntity.DataSource}, Url: {linkedEntity.Url.ToString()}, Entity Id in Data Source: {linkedEntity.DataSourceEntityId}");
     foreach (LinkedEntityMatch match in linkedEntity.Matches)
     {
-        Console.WriteLine($"    Match Text: {match.Text}, Score: {match.Score:0.00}, Offset: {match.GraphemeOffset}, Length: {match.GraphemeLength}.");
+        Console.WriteLine($"    Match Text: {match.Text}, Score: {match.Score}, Offset: {match.GraphemeOffset}, Length: {match.GraphemeLength}.");
```

Comment:
Unsure if you want to do this in dot net, but for Python we got rid of the mentions of grapheme length in samples and readme because of the uncertainty we discussed in scrum

Reply:
Hmm good point. Will remove from here and samples